eZ/Bundle/EzPublishCoreBundle/Tests/Translation/CollectorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * File containing the CollectorTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Tests\Translation;
+
+use eZ\Bundle\EzPublishCoreBundle\Translation\Collector;
+
+class CollectorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCollect()
+    {
+        $translationRootDir = str_replace('/Tests/Translation', '/Tests/Resources/Translation/vendor', __DIR__);
```

Comment:
Thank you for the appveyor pointer, @alongosz !

About vfs, after talking/thinking about it a bit, I'm not sure it has a lot of value in that particular context. It is very valuable when tests need to _write_ something, but not that much when you only need to read.